roles/foreman-proxy/defaults/main.yaml

```diff
@@ -0,0 +1,2 @@
+---
+foreman_proxy_image: quay.io/evgeni/foreman-proxy-rpm:3.12
```

Comment:
I recall we were saying we would avoid putting smart-proxy in a container to start with. I assume you added this because you need it for Pulp to work?

Reply:
Correct, it is essentially a stub that provides the config to katello. Probably should redesign this as part of this effort?

Reply:
How should we track that? An issue here? Or did you mean re-design more broadly?

Reply:
More broadly.
Today Katello talks directly to pulp (using the Ruby bindings), also on proxies.
But it does assume `smart_proxy_pulp` is present to derive URLs etc from it.
I find having a proxy plugin for a few strings a bit much :)

Reply:
I'll let you discuss that one with @ekohl :)

Reply:
When I talked with @ianballou he was thinking about always talking through `smart_proxy_pulp`.